src/MoBi.Presentation/Tasks/SerializationTask.cs

```diff
@@ -194,8 +194,6 @@ private string getElementNameFromBuildConfiguration(string expectedElementName)
          if (expectedElementName.Equals(AppConstants.XmlNames.MoleculeBuildingBlock)) return AppConstants.XmlNames.Molecules;
          if (expectedElementName.Equals(AppConstants.XmlNames.ObserverBuildingBlock)) return AppConstants.XmlNames.Observers;
          if (expectedElementName.Equals(AppConstants.XmlNames.EventGroupBuildingBlock)) return AppConstants.XmlNames.EventGroups;
-         if (expectedElementName.Equals(AppConstants.XmlNames.InitialConditionsBuildingBlock)) return AppConstants.XmlNames.InitialConditions;
-         if (expectedElementName.Equals(AppConstants.XmlNames.ParameterValuesBuildingBlock)) return AppConstants.XmlNames.ParameterValues;
```

Comment:
I don't understand why PArameterValues was removed? it's not supoorted anymore? dojn't we have a conversion from ParameterValues into somethihng?